packages/cli/src/commands/split/index.ts

```diff
@@ -237,49 +242,62 @@ function gatherComponentsFiles(
   componentsFiles[componentType][componentName] = { inherits, filename };
 }
 
-function iteratePaths(
-  openapi: Oas3Definition,
-  pathsDir: string,
-  openapiDir: string
+function iteratePathItem(
+  openapi: any,
```

Comment:
please specify a type

Reply:
It was renamed because ow iterates on webhooks also. The common thing is that both paths and webhooks has path item inside. 
customDir is used in tests right now - https://github.com/Redocly/openapi-cli/blob/split-webhooks/packages/cli/src/commands/split/__tests__/index.test.ts#L34, as I didn't come up with a better solution there.

Reply:
> please specify a type, There should be something better then `any` ;)

We only pass there an `openapi` parameter which can be of 2 types only:  `Oas3Definition | Oas3_1Definition`;
Duplicating those or having `object` type produces errors here - https://github.com/Redocly/openapi-cli/blob/split-webhooks/packages/cli/src/commands/split/index.ts#L252

Reply:
> The common thing is that both paths and webhooks has path item inside.

then rename it to `iteratePathItems`.

So how about changing the function accept path times object as a first arg and outDir as the second and call it like:

```
iteratePathItem(openapi.paths, path.join(openapiDir, 'paths'), componentsFiles);
iteratePathItem(openapi.webhooks || openapi['x-webhooks'] , path.join(openapiDir, 'webhooks'), componentsFiles);

Reply:
> Also two extra args seem to complex. What is customDir? looks like it's not used. How about changing the function accept paths object as a first arg and outDir as the second and call it like:
> 
> ```
> iteratePathItem(openapi.paths, path.join(openapiDir, 'paths'), componentsFiles);
> iteratePathItem(openapi, path.join(openapiDir, 'webhooks'), componentsFiles);
> ```

I can agree on the `customDir` part, but `rootDocObj` value is used in several places to define to logic between the webhooks and paths, mostly because of the different paths (e.g path starts with slash, webhooks don't). 
I believe the func itself is cleaner with aliases WEBHOOKS or PATHS, avoiding the actual strings.
Let me know if it makes sense.